### PR TITLE
Add HugePages config to kubernetes-worker charm docs

### PR DESCRIPTION
**Pull request by evilnick**
Adds section on config to charm-kubernetes-worker
Adds info on HugePages
Adds links to charm-kubernetes-master for IPVS

### CODE REVIEW

**Comment by evilnick** [comment]
For convenience: https://deploy-preview-421--cdk-docs-next.netlify.app/kubernetes/docs/charm-kubernetes-worker#k8s-services

File: pages/k8s/charm-kubernetes-worker.md [hunk 1]
```diff
@@ -265,6 +268,77 @@ the setting of conntrack-max-per-core vs nf_conntrack_max.
...
+[upstream documentation](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/)).
+
+To use HugePages in your pods with **Charmed Kubernetes**, it is necessary to
+update the configuration for the workers:
+
+1.  Fetch the current 'sysctl' configuration from the worker:
+    ```bash
+    juju config kubernetes-worker sysctl
+    ```
+    This should return a sting of config options, e.g.:
```
Comment (tvansteenburgh) [inline pages/k8s/charm-kubernetes-worker.md:295]: string

File: pages/k8s/charm-kubernetes-worker.md [hunk 2]
```diff
@@ -265,6 +268,77 @@ the setting of conntrack-max-per-core vs nf_conntrack_max.
...
+            hugepages-2Mi: 100Mi
+            memory: 100Mi
+          requests:
+            memory: 100Mi      
+      volumes:
+      - name: hugepage
+        emptyDir:
+          medium: HugePages  
+    ```
+    &nbsp;
```
Comment (tvansteenburgh) [inline pages/k8s/charm-kubernetes-worker.md:336]: In the rendered output this appears to add an additional unnecessary blank line.

Comment (evilnick) [reply pages/k8s/charm-kubernetes-worker.md:336]: the intentional unnecessary blank line is the one amazing trick I discovered to make the preceding YAML block to render correctly, after literally hours of research into code blocks in numbered indented lists.
it doesn't have to be blank though, perhaps I can add a final comment instead to make it look better.

File: pages/k8s/charm-kubernetes-worker.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/charm-kubernetes-worker.md:295]: thanks

**Pull request merged**